content/en/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks.md

```diff
@@ -0,0 +1,106 @@
+
+3. You can see an auto-generated link shown in **Remote Trigger Link**. Copy the contents of `/s2itrigger/v1alpha1/general/namespaces/demo-project/s2ibuilders/felixnoo-s2i-sample-latest-zhd/` as it will be included in the S2I webhook URL.
+
+   ![s2i-trigger-link](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/s2i-trigger-link.png)
+
+4. Log in to your GitHub account and go to the source code repository used for the S2I Image Builder. Go to **Webhooks** under **Settings** and then click **Add webhook**.
+
+   ![click-add-webhook](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/click-add-webhook.png)
+
+5. In **Payload URL**, enter `http://<IP>:<Service NodePort>/s2itrigger/v1alpha1/general/namespaces/demo-project/s2ibuilders/felixnoo-s2i-sample-latest-zhd/`. You can select trigger events based on your needs and then click **Add webhook**. This tutorial chooses **Just the push event** for demonstration purposes.
+
+   ![add-payload-url](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/add-payload-url.png)
+
+   {{< notice note >}}
+
+   `<IP>` is your own IP address, `<Service NodePort>` is the NodePort you get in step 1, and `/s2itrigger/v1alpha1/general/namespaces/demo-project/s2ibuilders/felixnoo-s2i-sample-latest-zhd/` is from the S2I remote trigger link. Make sure you use your own IP, Service NodePort and S2I remote trigger link. You may also need to configure necessary port forwarding rules and open port in your security groups depending on where your Kubernetes clusters are deployed.
+
+   {{</ notice >}}
+
+6. Once the webhook is added, you can click the webhook to view delivery details in **Recent Deliveries**. You can see a green tick if the Payload URL is valid.
+
+   ![webhook-delivery](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/webhook-delivery.png)
+
+7. After you finish all the above operations, the S2I Image Builder will be automatically triggered if there is a push event to the source code repository.
+
+   ![s2i-auto-build](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/s2i-auto-build.png)
+
+## Configure a B2I Webhook
+
+The method of configuring B2I webhook is the same as configuring S2I webhook.
+
+1. Expose the S2I trigger Service.
+
+2. View the **Remote Trigger Link** in the details page of your B2I Image Builder.
+
+   ![b2i-trigger-link](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/b2i-trigger-link.png)
+
+3. Add the payload URL in the source code repository. The B2I payload URL format is the same as that of S2I payload URL.
+
+   ![b2i-payload-url](/images/docs/project-user-guide/image-builder/s2i-and-b2i-webhooks/b2i-payload-url.png)
+
+   {{< notice note >}}
+
+   You may need to configure necessary port forwarding rules and open port in your security groups depending on where your Kubernetes clusters are deployed.
```

Comment:
open port in your security groups depending on where your Kubernetes clusters are deployed.
->
open the port in your security groups depending on where your Kubernetes cluster is deployed.